scripts/configure-installer-tests-cluster.sh

```diff
@@ -39,8 +39,100 @@ if [ -z $CI ]; then
 fi
 
 # Setup the cluster for Operator tests
-## First, enable a cluster-wide operator
-oc create -f - <<EOF
+
+## Create a new namesapce which will be used for OperatorHub checks
+oc new-project $CI_OPERATOR_HUB_PROJECT
+## Let developer user have access to the project
+oc adm policy add-role-to-user edit developer
+
+CI_SERVER_VERSION=$(oc version | awk '/Server/ {print $3}')
+
+## If we're running on 4.1, perform relevant steps
+
+if [ $CI_SERVER_VERSION == 4.1.0 ]; then
```

Comment:
what about other 4.x versions?

Reply:
Right. I don't know why this line registered as `4.1.0` in my mind https://github.com/openshift/release/blob/master/ci-operator/config/openshift/odo/openshift-odo-master__v4.1.yaml#L13. It's just `4.1`. I think I'll just change that `[[ $CI_SERVER_VERSION =~ 4.1 ]]`.